src/retroshare/rsconfig.h

```diff
@@ -395,20 +395,24 @@ class RsServerConfig
 	/**
 	 * @brief SetMaxDataRates set maximum upload and download rates
 	 * @jsonapi{development}
-	 * @param[in] downKb download rate in kB
-	 * @param[in] upKb upload rate in kB
+	 * @param[in] inKb download rate in kB
+	 * @param[in] outKb upload rate in kB
+	 * @param[in] inKbWhenIdle download rate in kB When Idle
+	 * @param[in] outKbWhenIdle upload rate in kB When Idle
 	 * @return returns 1 on succes and 0 otherwise
 	 */
-    virtual int SetMaxDataRates( int downKb, int upKb ) = 0;
+    virtual int SetMaxDataRates( int inKb, int outKb, int inKbWhenIdle, int outKbWhenIdle ) = 0;
```

Comment:
This will break API retro-compatibility, what's worse is that JSON API user will not even notice, they will just get random data rates set.

In this case I see two possibilities, 
1)  Provide a new method
`virtual int setMaxDataRates( int inKb, int outKb, int inKbWhenIdle, int outKbWhenIdle ) = 0;`
In this case as the former is not coherent with naming convention you can even avoid the V2 at the end of the name thanks to that sS (we cannot expose method with same name in JSON API) and mark the former `SetMaxDataRates` as deprecated

2) Provide a method to just set the idle rate and leave `SetMaxDataRates` as is without even touching parameters names

AFAIU 1 is what make more sense but i am not 100% sure